tests/remote/remote_controller.pm

```diff
@@ -19,6 +19,8 @@ sub run() {
     my $target_ip;
 
     select_console 'root-console';
+    type_string "chown $username /dev/$serialdev\n";
```

Comment:
shouldn't this be an `assert_script_run` instead of a `type_string`?

if it doesn't work, you probably don't want to continue, or do you?